var/spack/repos/builtin/packages/llvm/package.py

```diff
@@ -351,6 +357,12 @@ def cmake_args(self):
             ):
                 cmake_args.append("-DCMAKE_BUILD_WITH_INSTALL_RPATH=1")
 
+        if spec.satisfies('~code_signing'):
+            if not spec.satisfies('platform=darwin'):
+                return
```

Comment:
Why are you returning here? Then there won't be any CMake flags added for Linux.